docs/development/overview.md

```diff
@@ -28,7 +28,7 @@ performs well.
 Zulip also supports a wide range of ways to install the Zulip
 development environment:
 
-* On **Ubuntu** 18.04 Bionic and 16.04 Xenial and **Debian** 9
+* On **Ubuntu** 20.04 Focal and 18.04 Bionic and 16.04 Xenial and **Debian** 9
```

Comment:
I think this should have commas instead of so many `and`s.

Reply:
yes, agreed.

Reply:
Thanks for pointing it out. Fixed!!

Reply:
@timabbott any more changes?